app/code/Magento/Ui/Model/Export/MetadataProvider.php

```diff
@@ -118,13 +118,6 @@ public function getHeaders(UiComponentInterface $component)
         foreach ($this->getColumns($component) as $column) {
             $row[] = $column->getData('config/label');
         }
-
```

Comment:
Hi @shikhamis11. The lines removed by this PR were introduced to fix the order ID processing for archived orders and system data collector (Commerce Edition). By removing them we will bring up an issue that has been previously fixed. 
Instead of removing the piece of code, we need to adjust it to apply only for archived orders. Unfortunately, it can be tested only on Magento Commerce edition.
Anyway, we can't simply remove this code or remove double quotes from all orders.

Reply:
@rogyar: this sounds like a pretty terrible solution for those archived orders. Is there no way that can be fixed in a more elegant way in a module in Magento Commerce itself?